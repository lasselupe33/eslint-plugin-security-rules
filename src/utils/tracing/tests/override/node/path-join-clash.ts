import path from "path";

const arr = [1, 2, 3];
const pth = "myPath.ts";

const start = `${path.join(__dirname, pth)}-${arr.join("-")}`;

export {};
