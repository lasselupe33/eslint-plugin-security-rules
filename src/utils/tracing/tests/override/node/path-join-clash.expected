__dirnamemyPath.ts-123